syntax = "proto3";

message UserLogin{
    string login=1;
    string password=2;
}

enum UserAction {
    JOINED=0;
    LEAVED=1;
    UPDATED=2;
}

message UserInfo{
    string login=1;
}

message UserActionInfo {
    UserAction action=1;
    UserInfo user=2;   
}